src/mod/endpoints/mod_sofia/sofia.c

```diff
@@ -2949,6 +2949,9 @@ void *SWITCH_THREAD_FUNC sofia_profile_worker_thread_run(switch_thread_t *thread
 
 	sofia_set_pflag_locked(profile, PFLAG_WORKER_RUNNING);
 
+	/* Seed PRNG for functions within worker thread */
+	srand((unsigned)((intptr_t) switch_thread_self() + switch_micro_time_now()));
```

Comment:
you could use switch_time_now() instead of switch_micro_time_now() for better seed.

Reply:
Can you explain why? The seed is longer with micro.

Thanks for the review!

Reply:
it's just clock precision, switch_micro_time_now() does not increment that often practically as switch_time_now(). however, it seems it's fine like this.